Makefile

```diff
@@ -0,0 +1,18 @@
+CARTO = carto
+
+all: standard maxspeed signals electrification
+
+standard: project.xml
+
+maxspeed: maxspeed.xml
+
+signals: signals.xml
+
+project.xml: project.mml fonts.mss standard_labels.mss standard.mss standard_symbols.mss
+	$(CARTO) $< > $@
```

Comment:
Even if this is a rather trivial rule at the moment, I wonder if we should go for a general xml rule like ```%.xml:``` and just list the extra dependencies for the xml files. The standard rule can then depend on e.g. fonts.mss already.